python/tvm/meta_schedule/testing/custom_builder_runner.py

```diff
@@ -17,7 +17,7 @@
 """Customized builder and runner methods"""
 # pylint: disable=import-outside-toplevel
 
-from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Union
+from typing import TYPE_CHECKING, Dict, List, Any, Optional, Union, Callable
```

Comment:
remove `Callable`?

Reply:
It's used by argument `continuation` in `run_module_via_rpc`.

Reply:
sounds good. thanks for pointing this out!